libcxx/include/__format/format_functions.h

```diff
@@ -447,10 +449,41 @@ format_to(_OutIt __out_it, wformat_string<_Args...> __fmt, _Args&&... __args) {
 }
 #  endif
 
+template <class _CharT>
```

Comment:
Please add some documentation what this function does.
The code is not easy to understand and I don't think it does what I expect it to do.
Does the code optimize `std::format("{}", 42)`?

Reply:
I'm not quite sure what kind of documentation you're asking for. Do you want a comment listing the folds it can do?

> Does the code optimize `std::format("{}", 42)`?

Not currently. I plan to add more folds though.

Reply:
I find the code very hard to read and hard to understand what it tries to do. So I really like some comment in the code explaining what happens, that also would help me to review the code.